### DISCUSSION TITLE
插入usb移动硬盘容易假死

### DISCUSSION THREAD

**Discussion by yyshell** [immortalwrt/immortalwrt#1317] [Q&A]
我的imm是最新的快照版本，通过u盘启动系统，再加入一个移动硬盘时（低功耗mstata ssd盘），软路由会假死，luci界面进不去，但路由还在工作。
我的mstata ssd是ext4文件系统，系统有自动挂载的选项。
强制重启路由器后，可以正常挂载，也不死机了，但工作状态下，插入usb盘，还是会卡、假死。
这个是bug吗？

**Comment by initzhu** [comment]
什么设备 供电是否充足

**Reply by yyshell** [reply]
mstata的小ssd，超级功耗的，不存在供电问题。
